conf/comitup.conf

```diff
@@ -69,3 +69,13 @@
 # The primary device is used to spawn the access point.
 #
 # primary_wifi_device: wlan0
+
+# manage_wired_device
+#
+# By default, eth0 (wired) device is not managed by comitup, it is ignored.
+# By setting manage_wired_device, comitup will also monitor eth0 connection state.
```

Comment:
1) Make this "By _defining_ manage_wired_device ..."
2) "manage" is the wrong term. This change doesn't cause eth0 to be managed any differently. "monitor_wired_device"?